README.md

```diff
@@ -82,17 +82,19 @@ To watch the files
 yarn build:watch
 ```
 
-### Release a version
+### Releasing a version
 
-```
-yarn bump
-```
+#### Update the version
+Remember we follow [SemVer](http://semver.org/) rules.
 
-To publish a release to the `CDN` and `npm`
+    yarn bump:patch|minor|major
 
-```
-yarn release
-```
+#### Publishing the library
+Once you have your `secrets.json` file configured run
```

Comment:
I think we should remove this section entirely to be end-user centric and not so CARTO VL developer-centric, see https://github.com/CartoDB/carto-vl/issues/565